lib/windowsbasetest.pm

```diff
@@ -21,6 +21,11 @@ sub windows_run {
 
 
 sub _setup_serial_device {
+    # Prevent that openqa-agent blocks the serial port, if present
+    type_string "Stop-Service openqa-agent";
```

Comment:
Why not use `enter_cmd` or `enter_cmd_slow`?

Reply:
I follow the same procedures and later on in this subroutine.

Reply:
Ok I see now... Maybe we shall re-think it but for now, if it works don't touch it.